web/js/map/natural-events/event-track.js

```diff
@@ -57,10 +62,13 @@ const updateSelection = function(newDate) {
  * @param {Object} map OpenLayers map Object
  * @param {String} selectedDate
  * @param {Function} callback
+ * @param {Function} callbackHighlight
+ * @param {Function} callbackUnhighlight
  * @param {Function} showAllTracks
+ * @param {Boolean} isHighlighted
  * @return {Object} Object Containing track info and elements
  */
-const getTracksAndPoints = function (eventObj, proj, map, selectedDate, callback, showAllTracks) {
+const getTracksAndPoints = function (eventObj, proj, map, selectedDate, callback, callbackHighlight, callbackUnhighlight, showAllTracks, isHighlighted) {
```

Comment:
How difficult would it be to refactor this function to reduce the number of arguments? Something like 
```
const getTracksAndPoints = function (eventObj, proj, map, selectedDate, callback, showAllTracks, options) {
  const {
    callbackHighlight,
    callbackUnhighlight,
    isHighlighted,
  } = options
  ...
  ```
  9 arguments for a function just seems like it's getting a bit bloated. No big deal if it would be too much extra work though. IMO, functions with more than 3 arguments can get a little unwieldy to handle.